Makefile

```diff
@@ -6,5 +6,11 @@ check-licence:
 		find . -name "*.go") | sed -e 's,\./,,' | sort | uniq -u | \
 		xargs -I {} echo FAIL: licence missed: {}
 
-check: check-licence
+check-go:
+ifneq ($(strip $(shell gofmt -l .)),)
+	$(error go fmt is sad)
```

Comment:
1. Done
2. Fortunately gofmt -l . does, go fmt doesn't
3. I've added PHONY for all phony targets, also moved check to the top (so that it's default)

(depends on which side of the pacific are you :)